helm/oauth2-proxy/values.yaml

```diff
@@ -64,7 +72,9 @@ alphaConfig:
   # Arbitrary configuration data to append to the metrics section
   metricsConfigData: {}
   # Arbitrary configuration data to append
-  configData: {}
+  configData:
+    upstreams:
+      - "file:///dev/null"
```

Comment:
Why do we have to have it twice? Here and in the `legacyDefaults` block?

Reply:
Because here it's a line of yaml, in legacyDefaults it's a line of toml. The legacy file must be TOML and the alphaConfig is yaml. Without introducing a macro that generate TOML from YAML we have to have it twice.